extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-view.ts

```diff
@@ -235,6 +240,13 @@ export class DataExtensionsEditorView extends AbstractWebview<
 
         await Promise.all([this.setViewState(), this.loadExternalApiUsages()]);
 
+        break;
+      case "hideModeledApis":
+        await this.updateModelDetailsPanelState(
```

Comment:
```suggestion
        this.hideModeledApis = msg.hideModeledApis;
        await this.updateModelDetailsPanelState(
```

Needs to set `this.hideModeledApis` as well as updating the details panel state. Otherwise `this.hideModeledApis` stays set to the default value and whenever you update the state from one of the other locations (e.g. changing active data extensions editor) it'll set the state back to the default `hideModeledApis` value.